client/components/Channel.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+	<!-- TODO: investigate -->
```

Comment:
todo?

Reply:
as stated in IRC, no clue what the ref is supposed to be doing here (in master too). Accidentally left in?

Reply:
Something somewhere used $refs to the element. I saw there's a couple of querySelectors in client code, which likely needs to be replaced with refs.

https://vuejs.org/guide/essentials/template-refs.html

Reply:
I'll add that to TODO pr, but i dont see any references to refs.element / refs.wrapper on this branch or in master.